WorldState/WarframeStatusProvider.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace WorldState
+{
+    /// <summary>
+    /// A <see cref="WorldStateProvider"/> that uses online data from <a href="https://docs.warframestat.us/">docs.warframestat.us</a>.
```

Comment:
the docs are at docs.warframestat.us, but the data is at api.warframestat.us 🤔 sorry, the wording on the docs that mention it here are confusing me

Reply:
https://api.warframestat.us/ doesn't provide any useful information in a browser (besides not throwing me an error page). I put in https://docs.warframestat.us/ because I was thinking a link in the documentation was (kinda) supposed to link to another documentation.

It doesn't feel right linking to a blank page that implicitly tells people it works since the return code is 200. We know it's just a data source but it may cause some confusion before people starting realizing the page is only accessible via subroutine APIs.

I guess I could take out the `href` so people don't click on it. Personally I think it's better this way, linking to web API documentation.